### ISSUE TITLE
Map the source attributes only when the destination attributes are null 

### ISSUE THREAD

**Issue by GiritharanR** [modelmapper/modelmapper#230]
Model mapper to map the non null values of a destination. Is it possible?

It will help us to combine the Destination from two differnt sources.

```
Ex, 
Destination: 
Class A {
int i;
String s;
}

Source 1: 
Class B {
int i;
}

Source 2:
Class C {
String s;
}

I want to do something like below,
ModelMapper mapper = new Modelmapper();
A a = mapper.map(b, A.class);
A a1 = mapper.map(c, a);

At this point a1 should have both i and s values.
```

**Comment by corzar71** [comment]
You can use a custom PropertyMap to skip fields for every map you don't want by using the skip() method.

**Comment by chhsiao90** [comment]
@GiritharanR - Yeah, we had considered to introduce a configurable flag ```skipNull``` to skip the mapping action if the value is null. I had implement that at one of my [branch](https://github.com/chhsiao90/modelmapper/tree/skip-null) with [commit](https://github.com/chhsiao90/modelmapper/commit/756bfd7c0400235c8d14b0d242beaaf2b58cd0ff). 

I think this might solve your problem, but I need more unit test for this flag enabled before I merged it to master. So please wait me for a moment to merge this feature into our master branch!

Before that, I think you can try ```skip``` as @corzar71 mentioned I think that should work too!

@corzar71 - Thanks for the answer!

**Comment by chhsiao90** [comment]
@GiritharanR - we had the skipNull feature to be released with v1.1.0, please take a try!
